crm_stage_multi_team/hooks.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2025 Teacnativa - Eduardo Ezerouali
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import SUPERUSER_ID, api
+
+
+def post_init_hook(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    stages = env["crm.stage"].search([("team_id", "!=", False)])
+    for stage in stages:
+        if stage.team_id:
+            stage.write({"team_ids": [(6, 0, stage.team_id.ids)]})
```

Comment:
```suggestion
        stage.write({"team_ids": [(6, 0, stage.team_id.ids)]})
```

I believe the if condition is redundant.